src/cloud/project/new-relic.md

```diff
@@ -250,29 +249,27 @@ For further information and examples, see [Introduction to New Relic Logs][New R
 
 ### Monitor performance with Managed Alerts
 
-Adobe provides the Managed Alerts for {{ site.data.var.ee }} alert policy on Cloud Pro Production environments to track the following metrics:
-
--  [Apdex score]
--  error rate
--  disk space (available on Pro Production environments only)
--  CPU usage
--  memory usage
+Adobe provides the Managed Alerts for {{ site.data.var.ee }} alert policy to track performance metrics.
+Based on industry best practices, the policy includes a collection of alerts that set thresholds to trigger  warning and critical notifications when site infrastructure or application issues affect performance. The Managed Alerts policy tracks the following metrics on Production environments only:
 
-Based on industry best practices, this policy is a collection of alerts that set thresholds for warning and critical conditions that affect performance.
+| Metric | Data collection | Availability
+|:--------|:------------------------|:-------------
+| [Apdex score] | APM   | Pro and Starter
+| error rate    | APM   | Pro and Starter
+| disk space    | NRI   | Pro
+| CPU usage     | NRI   | Pro
+| memory usage  | NRI   | Pro
 
-When your site experiences an infrastructure or application issue that triggers an alert threshold, New Relic sends alert notifications so that you can proactively address the issue.  See [Managed Alerts for Magento Commerce][] in the _Magento Help Center_ for details about each alert type and troubleshooting steps to resolve the issues that triggered the alert.
-
-{:.bs-callout-warning}
-To use this policy and get notified when an issue on your site triggers an alert condition, you must configure at least one notification channel to receive the alert notifications.
+When site infrastructure or application conditions trigger an alert threshold, New Relic sends alert notifications so that you can proactively address the issue.  See [Managed Alerts for Magento Commerce] in the _Magento Help Center_ for details about alert thresholds and troubleshooting steps to resolve the issues that triggered the alert.
```

Comment:
```suggestion
When site infrastructure or application conditions trigger an alert threshold, New Relic sends alert notifications so that you can proactively address the issue. See [Managed Alerts for Magento Commerce] in the _Magento Help Center_ for details about alert thresholds and troubleshooting steps to resolve the issues that triggered the alert.
```